acceptance/tests/fixtures/cases/sameness/static-client/default/patch.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: static-client
+spec:
+  template:
+    metadata:
+      annotations:
+        'consul.hashicorp.com/connect-inject': 'true'
+        consul.hashicorp.com/connect-service-upstreams: "static-server.ns2.default:8080"
```

Comment:
~ More for my own understanding: is there a reason the first key is quoted, and the second one isn't? (same w/ other similar patch files)

Reply:
no idea actually 😓 these are from Eric's demo

Reply:
Cool, and totally not a blocker! Seems another is double-quoted, so probably just copy 🍝